Source/SteelSeriesGameSense/Private/SSGS_ClientPrivate.cpp

```diff
@@ -599,18 +600,24 @@ Client::_gsWorkerReturnType_ Client::_gsWorkerFn()
 
 Client* Client::Instance()
 {
+    if ( !_mpInstance ) {
+        LOG( Display, TEXT( "Creating new GameSense client instance" ) );
+        _mpInstance = new Client;
```

Comment:
This line doesn't have the std::nothrow that the one in Initialize had.  Is that something you meant to remove?

Reply:
Yes, actually did mean to remove that. We don't need to specify "nothrow" since exceptions aren't enabled. There's another commit that fixes stuff relating to 'noexcept' errors [HERE](https://github.com/SteelSeries/gamesense-plugin-ue4/pull/7/commits/79ca01ded3d8d5ac1810e9bec350637e536213ae)